actionpack/test/controller/render_test.rb

```diff
@@ -1,6 +1,5 @@
 require 'abstract_unit'
 require 'controller/fake_models'
-require 'pathname'
```

Comment:
@kaspth git grep Pathname -- actionpack/test/controller/render_test.rb returns nothing (and tests are green without it)